integration/image/save_test.go

```diff
@@ -52,9 +52,9 @@ func tarIndexFS(t *testing.T, rdr io.Reader) fs.FS {
 }
 
 func TestSaveCheckTimes(t *testing.T) {
-	t.Parallel()
-
 	ctx := setupTest(t)
+
+	t.Parallel()
```

Comment:
Haven't checked; can we check if parallel is already set? Wondering if our `setupTest()` should check for it, and panic, similar to how `t.Parallel()` itself also checks for that; https://github.com/golang/go/blob/b44f6378233ada888f0dc79e0ac56def4673d9ed/src/testing/testing.go#L1443-L1448

Reply:
I don't see an explicit way to check it.

A hacky way would be to either use reflection, or call `Setenv` (which panics when `Parallel` was already set) and `recover` it.

Reply:
Yeah, I may have searched for that as well in the past. Would've been nice, because if there was a way, we could make `setuptest()` panic if it was used incorrectly.

Reply:
I guess the alternative for now could also be to add a comment (`t.Parallel` must be called AFTER `setupTest()`); do you think it's worth adding?